clustered/integration-test/src/test/java/org/ehcache/clustered/replication/BasicClusteredCacheOpsReplicationTest.java

```diff
@@ -62,6 +68,16 @@
   private static Cache<Long, String> CACHE1;
   private static Cache<Long, String> CACHE2;
 
+  @Parameters(name = "consistency={0}")
```

Comment:
This can be simplified with

```java
  @Parameters
  public static Consistency[] data() {
    return Consistency.values();
  }

  @Parameter
  public Consistency cacheConsistency;
```